tsconfig.json

```diff
@@ -25,6 +25,9 @@
     "strictBindCallApply": false,
     "experimentalDecorators": true,
 
+    "skipLibCheck": true,
+    "isolatedModules": true,
```

Comment:
I don’t think you should use this inside of `tsconfig`. It is more of a setting that you use for `ts-jest` or any kind of bundler plugin. Depending on the project size (und this seems quite big), its a good idea to separate typechecks from other tasks. I will try to blog about this to better explain what I mean.

Reply:
I agree with this - I think we want to do typechecking of libs during development and we probably want a separate/dedicated CI job that does typechecking while the other jobs do not type check.